e2e/src/page-objects/pages/home.page.ts

```diff
@@ -35,8 +36,21 @@ export class HomePage extends Page {
     await findElement(
       AssetsSelectors.assetItemButton,
       { name },
-      SHORT_TIMEOUT,
+      VERY_SHORT_TIMEOUT,
       `${name} token not found in the token list on the Home page`
     );
   }
+
+  async isTokenNotDisplayed(name: string) {
+    await retry(
+      () =>
+        this.isTokenDisplayed(name).then(
+          () => {
+            throw new Error(`Token with slug: '${name}' not deleted/hidden and displayed on the Home page`);
```

Comment:
why do we throw Error in `then` but not in `catch`

Reply:
Because here is reverse logic. When we got positive result, it means for us negative, we should throw error, when result is negative, we should ignore another error by catch and test will passed